galite-tests/src/test/kotlin/org/kopi/galite/tests/form/VFieldTests.kt

```diff
@@ -377,4 +381,54 @@ class VFieldTests : JApplicationTestBase() {
 
     assertEquals(20, FormSample.tb4ToTestListDomain.age.value)
   }
+
+  @Test
+  fun `fetchColumn existing column scenario test` () {
+    val field = FormSample.tb1.blockFields[0].vField
+
+    assertEquals(0, field.fetchColumn(User))
+  }
+
+  @Test
+  fun `fetchColumn not existing column scenario test` () {
+    val field = FormSample.tb1.blockFields[4].vField
+
+    assertEquals(-1, field.fetchColumn(User))
+  }
+
+  @Test
+  fun `enter test valid scenario` () {
+    val field = FormSample.tb1.blockFields[3].vField
+
+    field.block?.enter()
+    field.block?.activeField?.leave(false)
+    field.enter()
```

Comment:
the test work these two lines

Reply:
The `.enter` method in VField requires that both the `activeField` is null and the field block is the same `activeBlock` of the form. Therefore i used the `.leave` method to nullify the activeField attribut